lib/Reply.js

```diff
@@ -26,9 +26,11 @@ class Reply {
     this.msg = {
       statements: [],
       reprompt: '',
+      hint: undefined,
```

Comment:
I'm not sure i like this, why does `hint` gets special treatment instead of just going in the `directives` array?

Reply:
i mean, i do like the fact you can declare it like this in the view. What i don't like is the fact that we don't treat it internally as just another directive